src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
```

Comment:
but the playback api never tells if its stopped, only if its playing or not, or? IIRC when you go afk for a longer period of time, the current playlist gets resetted so that the playbutton is grey again, or?